src/components/Quote.js

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+
+export default function Quote() {
+  const [loading, setLoading] = useState(true);
```

Comment:
- [OPTIONAL] I suggest that you set  loading to false by default and only set it to true inside the useEffect   hook when the fetch request is run.
   
```
useEffect(() => {
    const fetchQuotes = async () => {
      setIsLoading(true)
      try {
```
 This is because at the beginning it is not loading and therefore loading should be set to false 👍🏾